### PR TITLE
Removed unnecessary files

### PR DESCRIPTION
**Pull request by cfnyu**
This file is not needed

### CODE REVIEW

File: app/connection.py [hunk 1]
```diff
@@ -1,44 +0,0 @@
...
-        port = '5432'
-
-        raise ValueError(TESTING)
-
-        logging.info("Connecting to database on host %s port %s",
-                     hostname, port)
-
-        uri = 'postgres://{}:{}@{}:{}/{}'
-        uri = uri.format(username, password, hostname, port, db_name)
-
```
Comment (sfloam) [inline app/connection.py]: Do we no longer use this?

**Review by sfloam — APPROVED** [review]
Good to go.

**Comment by cfnyu** [comment]
IBM Cloud [toolchain](https://console.bluemix.net/devops/toolchains/b70ebc13-f2da-447f-9876-2a7d703481b6 "toolchain-nyu-recommendation-service-f17"): [Delivery Pipeline](https://console.bluemix.net/devops/pipelines/f3063f54-6e1d-4e45-8b7c-b27afa21a694 "delivery") deployed [recommendations](https://github.com/devopsdelta/recommendations/releases/tag/deploy-ibm-yp-us-south-nyu-delta-team-f17-dev-20171213-223932 "deploy-ibm-yp-us-south-nyu-delta-team-f17-dev-20171213-223932") to [dev](https://console.bluemix.net/apps/c7225f97-1211-46e5-9fbc-8559da2c5fe7?env_id=ibm:yp:us-south "ibm:yp:us-south:nyu-delta-team-f17:dev"), including commit a7cf5f4a7577b53e7c40f8bb33e60a5b5c5a7c5f

**Comment by cfnyu** [comment]
IBM Cloud [toolchain](https://console.bluemix.net/devops/toolchains/b70ebc13-f2da-447f-9876-2a7d703481b6 "toolchain-nyu-recommendation-service-f17"): [Delivery Pipeline](https://console.bluemix.net/devops/pipelines/f3063f54-6e1d-4e45-8b7c-b27afa21a694 "delivery") deployed [recommendations](https://github.com/devopsdelta/recommendations/releases/tag/deploy-ibm-yp-us-south-nyu-delta-team-f17-prod-20171213-225345 "deploy-ibm-yp-us-south-nyu-delta-team-f17-prod-20171213-225345") to [prod](https://console.bluemix.net/apps/f9460b21-5cc6-4470-b00b-9de89c5a370a?env_id=ibm:yp:us-south "ibm:yp:us-south:nyu-delta-team-f17:prod"), including commit a7cf5f4a7577b53e7c40f8bb33e60a5b5c5a7c5f

**Pull request merged**